mocha.js

```diff
@@ -1641,6 +1641,20 @@ Mocha.prototype.useInlineDiffs = function (inlineDiffs) {
 };
 
 /**
+ * Do not show diffs at all.
```

Comment:
please don't commit changes to this file.  it's lame we keep it under VCS (b/c bower)